FusionIIIT/applications/complaint_system/models.py

```diff
@@ -73,8 +73,8 @@ class StudentComplain(models.Model):
     complaint_type = models.CharField(choices=Constants.COMPLAINT_TYPE,
                                       max_length=20, default='internet')
     location = models.CharField(max_length=20, choices=Constants.AREA)
-    specific_location = models.CharField(max_length=50, blank=True)
-    details = models.CharField(max_length=100)
+    specific_location = models.CharField(max_length=100, blank=True)
+    details = models.TextField()

```

Comment:
@samay-rgb the field has been changed to TextField from Charfield.

Reply:
Looks fine since details can have over 255 characters.

Reply:
if this field is not required then use blank=true